code/modules/mob/new_player/server_info_window.dm

```diff
@@ -0,0 +1,58 @@
+/client/verb/check_server_info()
+	set name = "Server Information"
+	set category = "OOC"
+
+	if (src.infowindow)  //If the user's already viewed the window before just load it again.
+		src.infowindow.open()
+		return
+
+	var/list/close = list("Gotcha!","AFFIRMATIVE","EXCELSIOR!","I accept","The Pact Is Sealed","Continue","Take me to the fun!","Let us begin")
+	var/output = {"
+	[join_motd]
+	<br>
+	[file2text("config/rules.html")]
+	<p><b>Map info:</b></p>
+	[GLOB.using_map.map_info]
+	<br>
+	<br>
+	<div align='center'>
+	<strong>Useful links:</strong>
+	<br>
+	<b><a href='byond://?src=\ref[src];bugtracker_link=1'>Bugtracker</A></b>
+	<b><a href='byond://?src=\ref[src];wiki_link=1'>Wiki</A></b>
+	<b><a href='byond://?src=\ref[src];forum_link=1'>Forum</A></b>
+	<b><a href='byond://?src=\ref[src];changelog_link=1'>Changelog</A></b>
+	<br>
+	This server is running the [game_version] modification of <a href='byond://?src=\ref[src];upstream_link=1'>[config.upstream]</A>'s SS13 code.
+	<p><b><a href='byond://?src=\ref[src];close_info=1'>[pick(close)]</A></b></p>
+	</div>"}
+	infowindow = new(src.mob, "Welcome to [game_version]","Welcome to [game_version]", 768, 768, src)
+	infowindow.set_window_options("can_close=0")
+	infowindow.set_content(output)
+	infowindow.open()
+	return
+
+client/Topic(href, href_list[])
```

Comment:
General practice for Topic as I recall is to do `. = ..()` at the beginning, and return early if `.` is true. It handles some exploit checking.